cypress-tests/cypress.json

```diff
@@ -0,0 +1,16 @@
+{
+    "defaultCommandTimeout": 10000,
+    "viewportWidth": 1920,
+    "viewportHeight": 1080,
+
+    "env": {
+        "test_user_name": "admin",
+        "test_user_password": "admin",
+        "api_endpoint": "http://che-eclipse-che.172.19.20.205.nip.io/api/",
+        "root_url": "http://che-eclipse-che.192.168.0.104.nip.io",
```

Comment:
hello, I'm not sure these addresses can be hardcoded ?

Reply:
https://github.com/eclipse/che/pull/12848#discussion_r264118915